### PR TITLE
Create: Key map: Ergo42: biacco-winjp

### PR DESCRIPTION
**Pull request by shimesaba-type0**
Created biacco-winjp key map for Ergo42 keyboard. This key map is same as biacco key map.
The target environment for this key map, JIS layout on Windows 7 and Japanese IME only.

Could you please merge this if all no problem.

### CODE REVIEW

File: keyboards/ergo42/keymaps/biacco-winjp/config.h [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#ifndef CONFIG_USER_H
```
Comment (drashna) [inline keyboards/ergo42/keymaps/biacco-winjp/config.h]: If you could replace the include guard (the ifndef, define, and endif at the end) with just `#pragma once`? 

It's simpler, and less prone to user error, so if you could make this switch?

File: keyboards/ergo42/keymaps/biacco-winjp/config.h [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+#define CONFIG_USER_H
+
+#include "../../config.h"
```
Comment (drashna) [inline keyboards/ergo42/keymaps/biacco-winjp/config.h]: This line is not actually needed, and should be removed. 

Specifically, the build script automatically includes all of the config.h files, so this is not necessary.

File: keyboards/ergo42/keymaps/biacco-winjp/rules.mk [hunk 3]
```diff
@@ -0,0 +1,3 @@
+ifndef QUANTUM_DIR
```
Comment (drashna) [inline keyboards/ergo42/keymaps/biacco-winjp/rules.mk]: This block isn't actually needed.

It's a holdover from the old build system, and is no longer needed.

File: keyboards/ergo42/keymaps/biacco-winjp/config.h (hunk 1 above)
Comment (shimesaba-type0) [reply keyboards/ergo42/keymaps/biacco-winjp/config.h]: I see, I'll try it.

File: keyboards/ergo42/keymaps/biacco-winjp/rules.mk (hunk 3 above)
Comment (shimesaba-type0) [reply keyboards/ergo42/keymaps/biacco-winjp/rules.mk]: Thank you for your comment. I'll remove this file.

File: keyboards/ergo42/keymaps/biacco-winjp/config.h (hunk 1 above)
Comment (shimesaba-type0) [reply keyboards/ergo42/keymaps/biacco-winjp/config.h]: I've changed in this commit 62da35313da0429cd180f674cad9523664bef05b

File: keyboards/ergo42/keymaps/biacco-winjp/config.h (hunk 2 above)
Comment (shimesaba-type0) [reply keyboards/ergo42/keymaps/biacco-winjp/config.h]: I've changed in this commit 62da35313da0429cd180f674cad9523664bef05b

File: keyboards/ergo42/keymaps/biacco-winjp/rules.mk (hunk 3 above)
Comment (shimesaba-type0) [reply keyboards/ergo42/keymaps/biacco-winjp/rules.mk]: I've changed in this commit 62da35313da0429cd180f674cad9523664bef05b
`rules.mk` file was removed.

**Comment by shimesaba-type0** [comment]
I've changed this request following your comment. (I think...) Could you please check this. And please merge this if all no problem.

File: keyboards/ergo42/keymaps/biacco-winjp/keymap.c [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+// #include <sendstring_jis.h>  // macro sendstring for jis keyboard マクロ文字列送信時に日本語キーボード設定での文字化け回避>
+
+extern keymap_config_t keymap_config;
+
+#define BASE 0
+#define META 1
+#define SYMB 2
+#define GAME 3
+
+// Fillers to make layering more clear
```
Comment (noroadsleft) [inline keyboards/ergo42/keymaps/biacco-winjp/keymap.c]: Lines 16-18 aren't needed here, as QMK defines these on every keymap.

https://github.com/qmk/qmk_firmware/blob/35e76539e7257a1932ec76882892b999e21c9d4f/quantum/quantum_keycodes.h#L29-L31

**Comment by shimesaba-type0** [comment]
Hi @drashna , I've fixed 3 your comments. Could you please re-check and commit if all OK.

**Comment by drashna** [comment]
Awesome, thanks!

**Comment by shimesaba-type0** [comment]
Thank you so much!

**Pull request merged**